test/Microsoft.Azure.WebJobs.Extensions.Kafka.EndToEndTests/KafkaEndToEndTests.cs

```diff
@@ -305,7 +305,7 @@ public async Task Multiple_Hosts_Process_Events_At_Least_Once()
                     return host1Events.Count > 0 &&
                         host2Events.Count > 0 &&
                         host2Events.Count + host1Events.Count >= producedMessagesCount;
-                });
+                }, 120 * 1000);
```

Comment:
Please revert this change, I have fixed the e2e tests.

Reply:
Done!